lib/schema_plus/indexes/middleware/schema.rb

```diff
@@ -8,7 +8,7 @@ def after(env)
             indexes = Hash[env.index_definitions.map{ |d| [d.name, d] }]
 
             env.connection.exec_query("SELECT name, sql FROM sqlite_master WHERE type = 'index'").map do |row|
-              if (desc_columns = row['sql'].scan(/['"`]?(\w+)['"`]? DESC\b/).flatten).any?
+              if row['sql'] && (desc_columns = row['sql'].scan(/['"`]?(\w+)['"`]? DESC\b/).flatten).any?
```

Comment:
@ronen I'm not completely sure why the query returns auto-generated indexes here.